src/olympia/amo/decorators.py

```diff
@@ -102,18 +102,14 @@ def wrapper(*args, **kw):
     json.dumps(s), content_type='application/json')
 
 
-def use_master(f):
+def use_primary_db(f):
     @functools.wraps(f)
     def wrapper(*args, **kw):
         with context.use_master():
```

Comment:
Could you change that context manager as well? It's defined in `olympia.amo.models`.

Reply:
And then you can also update `src/olympia/translations/tests/test_models.py` to import and use that context manager instead of `multidb.pinning.use_master`